config/locales/server.en.yml

```diff
@@ -1838,15 +1838,15 @@ en:
 
     enable_discord_logins: "Allow users to authenticate using Discord?"
     discord_client_id: 'Discord Client ID (need one? visit <a href="https://discordapp.com/developers/applications/me">the Discord developer portal</a>)'
-    discord_secret: "Discord Secret Key"
+    discord_secret: "Discord Client Secret Key Used for authenticating and enabling Discord related features on the site, such as Discord logins. This secret key corresponds to the Discord application created for the website, and is necessary for securely communicating with the Discord API. Misuse may lead to unauthorized access so handle it carefully."
```

Comment:
```
Misuse may lead to unauthorized access so handle it carefully.
```

I'm not sure what this means... feels unnecessary.

Reply:
Yep will remove